src/class-metadata.php

```diff
@@ -325,7 +325,7 @@ private function get_category_name( WP_Post $post_obj, array $parsely_options ):
 		 *
 		 * @param string  $category        Category name.
 		 * @param WP_Post $post_obj        Post object.
-		 * @param array   $parsely_options The Parsely options.
+		 * @param array<string, mixed>   $parsely_options The Parsely options.
```

Comment:
Spacing can probably be reduced here (1 space instead of 3 in the last line, and align the above items.